Makefile

```diff
@@ -57,7 +57,7 @@ sync:
 .PHONY: clean
 clean:
 	go clean -i ./...
-	rm -rf $(BIN) $(DIST)
+	rm -rf $(BIN) $(DIST) assets
```

Comment:
not sure about this. but since using a symllink to phoenix/dist folder does not work anyway we can get rid of the assets as well ...

Reply:
symlink? Where?

Reply:
This change is totally fine to reset the assets folder which is ignored via .gitignore.